README.md

```diff
@@ -27,10 +27,9 @@ After you're finished please remove all the comments and instructions!
 
 <div align="center">
   <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
-  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
   <br/>
 
-  <h3><b>Microverse README Template</b></h3>
+  <h3><b>School Library</b></h3>
```

Comment:
- [ ] Please update your README file following the [Microverse README Template](https://github.com/microverseinc/readme-template)
- [ ] Please do not put the project name in the position of the logo.

- [ ] Please remove the microverse logo **Optional** You can put yours if you have one.